includes/core-changes/serialization/5.0/deserializing-json-into-char-requires-single-character.md

```diff
@@ -0,0 +1,37 @@
+### JsonSerializer.Deserialize requires single-character string
+
+When the type parameter is <xref:System.Char>, the string argument to <xref:System.Text.Json.JsonSerializer.Deserialize%60%601(System.String,System.Text.Json.JsonSerializerOptions)?displayProperty=nameWithType> must contain a single character for deserialization to succeed.
+
+#### Change description
+
+In previous .NET versions, if you pass a multi-character string to <xref:System.Text.Json.JsonSerializer.Deserialize%60%601(System.String,System.Text.Json.JsonSerializerOptions)?displayProperty=nameWithType> and the type parameter is <xref:System.Char>, the deserialization succeeds and only the first character is deserialized.
+
+In .NET 5.0 and later, when the type parameter is <xref:System.Char>, passing anything other than a single-character string causes a <xref:System.Text.Json.JsonException> to be thrown.
```

Comment:
NIT: same
```csharp
    // Before (5.0): Returns the first character - 'a'
    // After (5.0): throw JsonException because payload has more than one character
    JsonSerializer.Deserialize<char>("abc");

    // Correct usage
    JsonSerializer.Deserialize<char>("a");
```

Reply:
You are missing quotes in the input. Just the string `a` without quotes surrounding it, is invalid JSON.